app/src/pages/authoring/[resourceType]/[id].tsx

```diff
@@ -58,21 +70,55 @@ export default function ResourceAuthoringPage() {
     }
   });
 
-  function parseUpdate(url: string, identifier: string) {
-    const update: { url?: string; identifier?: fhir4.Identifier[] } = {};
+  function parseUpdate(url: string, identifier: string, name: string, title: string, description: string) {
+    const additions: {
+      url?: string;
+      identifier?: fhir4.Identifier[];
+      name?: string;
+      title?: string;
+      description?: string;
+    } = {};
+
+    const deletions: {
+      url?: string;
+      identifier?: fhir4.Identifier[];
+      name?: string;
+      title?: string;
+      description?: string;
+    } = {};
+
     if (url !== '') {
-      update['url'] = url;
+      additions['url'] = url;
+    } else {
+      deletions['url'] = '';
     }
     if (identifier !== '') {
       const splitIden = identifier.split('|');
       if (splitIden.length > 1) {
-        update['identifier'] = [{ system: splitIden[0], value: splitIden[1] }];
+        additions['identifier'] = [{ system: splitIden[0], value: splitIden[1] }];
       } else {
-        update['identifier'] = [{ value: splitIden[0] }];
+        additions['identifier'] = [{ value: splitIden[0] }];
       }
+    } else {
+      deletions['identifier'] = [{ system: '', value: '' }];
```

Comment:
If the user is allowed to add more than one identifier using `"|"`, I think it might be helpful to emphasize this with the UI. Maybe you could add a tooltip note to the `identifier TextInput` to let the user know how to add multiple types of identifiers. Another idea might be to break this `identifier TextInput` into two text inputs: `A system identifier TextInput `and a `value identifier TextInput`.  Also that way you could display both of the chosen identifiers rather than just one?

Reply:
So I think that this is definitely something we are going to look into in the future, but for now I think it's out of scope for this task since this is how we already had it. See the FHIR spec [here](http://hl7.org/fhir/R4/datatypes.html#identifier) to see how identifier is defined since it's technically not more than one identifier. No need to worry too much about this right now though :)

Reply:
I still think this input could use an update though, at least with a tooltip to say that `system|version` is supported for a single identifier. I'd say it's in-scope of this task to update the text field to indicate supported formats

Reply:
Gotcha! I spoke to Matt and we decided it made the most sense to just have the identifier value and the identifier system be different inputs. The identifier system is optional, so it is only saved on the artifact JSON if the identifier value exists.